src/implementations/business/volatile/VolatileStorageSchemaProvider.ts

```diff
@@ -0,0 +1,47 @@
+import {
```

Comment:
The storage schema for the DB needs to be provided via the CrunchDB constructor; it's not something that will be configured or known before you initialize the CrunchDB